content/4_Gold/Sliding_Window.problems.json

```diff
@@ -192,6 +192,18 @@
     }
   ],
   "queue": [
+    {
+      "uniqueId": "cc-BINLAND",
+      "name": "Binary Land",
+      "url": "https://www.codechef.com/problems/BINLAND",
+      "source": "CC",
+      "difficulty": "Normal",
```

Comment:
why is this normal difficulty when it is the hardest problem in the module imo 🤔

Reply:
hm, can you stick to suggesting only problems that you've solved?